src/App.tsx

```diff
@@ -1,9 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [searchValue, setSearchValue] = useState('');
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchValue(prev => {
+      const current = event.target.value;
+
+      if (prev.length > 0) {
+        if (prev[prev.length - 1] === ' '
+          && current[current.length - 1] === ' ') {
+          return prev;
+        }
+      }
```

Comment:
It's hard to understand what this code actually does.

It would be great if I typed `Love              Actually` in the input and saw this film in the list